app/latigo/metadata_api/data_structures.py

```diff
@@ -39,8 +39,8 @@ class OutputTag:
     name: str
     time_series_id: str
     type: str
-    description: str = None
-    derived_from: str = None
+    description: Optional[str] = ""
+    derived_from: Optional[str] = ""
```

Comment:
`Optional` means that it may be `None`. Is it intended? I suppose it is always `str`:
```suggestion
    description: str = ""
    derived_from: str = ""
```

Reply:
nice catch, I'll roll logic in other way

Reply:
done